tests/simulation/test_simulator.py

```diff
@@ -194,3 +196,83 @@ def test_compare_mempool_implementations(self):
                 set(tx_storage.iter_mempool_from_tx_tips()),
             )
             self.simulator.run(10)
+
+
+class MixedSyncRandomSimulatorTestCase(SimulatorTestCase):
```

Comment:
Should we move this test to a separate test file, maybe in the `tests/p2p/test_sync_bridge.py`?

Reply:
I'll do that.

Reply:
Done! Thanks!